test/System/Vector2.cpp

```diff
@@ -244,76 +243,76 @@ TEST_CASE("sf::Vector2 class template - [system]")
     {
         const sf::Vector2f v(2.4f, 3.0f);
 
-        CHECK(v.length() == Approx(3.84187));
-        CHECK(v.lengthSq() == Approx(14.7599650969));
-        CHECK(v.normalized() == ApproxVec2(0.624695f, 0.780869f));
+        CHECK(v.length() == Approx(3.84187f));
+        CHECK(v.lengthSq() == Approx(14.7599650969f));
+        CHECK(v.normalized() == Approx(sf::Vector2f(0.624695f, 0.780869f)));
 
         const sf::Vector2f w(-0.7f, -2.2f);
         
-        CHECK(w.length() == Approx(2.30868));
-        CHECK(w.lengthSq() == Approx(5.3300033));
-        CHECK(w.normalized() == ApproxVec2(-0.303204f, -0.952926f));
+        CHECK(w.length() == Approx(2.30868f));
+        CHECK(w.lengthSq() == Approx(5.3300033f));
+        CHECK(w.normalized() == Approx(sf::Vector2f(-0.303204f, -0.952926f)));
     }
 
     SUBCASE("Rotations and angles")
     {
         const sf::Vector2f v(2.4f, 3.0f);
         
-        CHECK(v.angle() == ApproxDeg(51.3402f));
-        CHECK(sf::Vector2f::UnitX.angleTo(v) == ApproxDeg(51.3402f));
-        CHECK(sf::Vector2f::UnitY.angleTo(v) == ApproxDeg(-38.6598f));
+        CHECK(v.angle() == Approx(sf::degrees(51.3402f)));
+        CHECK(sf::Vector2f::UnitX.angleTo(v) == Approx(sf::degrees(51.3402f)));
+        CHECK(sf::Vector2f::UnitY.angleTo(v) == Approx(sf::degrees(-38.6598f)));
```

Comment:
What do you think of using the `_deg` notation here (and in similar places)?

Reply:
Sounds good to me. I'll use `_deg` pervasively throughout that file since it already has `using namespace sf::Literals` at the top.